cypress/support/find.js

```diff
@@ -16,11 +16,11 @@ import { parseSelectorWithDataTest } from './common/parseSelectorWithDataTest.js
  * cy.get('.foo').find('{node} input', 'custom-prefix')
  * cy.get('.foo').find('[data-test="custom-prefix-node"] input')
  */
-// eslint-disable-next-line max-params
-const find = (originalFn, subject, selectors, options = {}) => {
-    const { prefix, ...restOptions } = options
-    const selector = parseSelectorWithDataTest(selectors, prefix)
-    return originalFn(subject, selector, restOptions)
-}
-
-Cypress.Commands.overwrite('find', find)
+Cypress.Commands.overwriteQuery(
```

Comment:
do you know why did we create this overwrite in the first place? seems a bit obscure/non-standard .. are we actively relying on it?

Reply:
I'll create a ticket for this

Reply:
https://dhis2.atlassian.net/issues/LIBS-607